lib/Transforms/Scalar/LowerTypePasses.cpp

```diff
@@ -223,6 +223,7 @@ void DynamicIndexingVectorToArray::ReplaceStaticIndexingOnVector(Value *V) {
         // Skip the pointer idx.
         Idx++;
         ConstantInt *constIdx = cast<ConstantInt>(Idx);
+        Type *Ty = GEP->getType()->getPointerElementType();
```

Comment:
If only use is in CreateAlloca, why not just do it there?

Reply:
The GEP could used by more than one CallInst, so put the type out of the loop. Maybe should create Alloca here always, then remove if not used?